pom.xml

```diff
@@ -1,158 +1,166 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
          xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
-    <modelVersion>4.0.0</modelVersion>
-    <groupId>org.btools</groupId>
-    <artifactId>brouter</artifactId>
-    <version>1.5.5</version>
-    <packaging>pom</packaging>
-    <url>http://brouter.de/brouter/</url>
-    <name>brouter</name>
-    <description>configurable OSM offline router with elevation awareness, Java + Android</description>
+  <modelVersion>4.0.0</modelVersion>
+  <groupId>org.btools</groupId>
+  <artifactId>brouter</artifactId>
+  <version>1.5.5</version>
+  <packaging>pom</packaging>
+  <url>http://brouter.de/brouter/</url>
+  <name>brouter</name>
+  <description>configurable OSM offline router with elevation awareness, Java + Android</description>

```

Comment:
Hier in der Datei sind viele line-alignment (white space) changes.
Prinzipiell ist das unnötig die "fremde" Datei mit unseren Rules zu auto-formatieren aber ist jetzt auch nicht wild.
=> Kein change request.

Reply:
Ja. Die Formatierung folgt sowieso keiner Regel. Daher hab ich jetzt einfach mal unsere Regel angewandt. Die können sich ja beschweren, wenn sie den PR akzeptieren wollen.